controllers/secure.go

```diff
@@ -103,6 +105,17 @@ func (c *SecureContext) submitRequest(rw http.ResponseWriter, req *http.Request,
 	if contentHeader := req.Header.Get("Content-Type"); len(contentHeader) > 0 {
 		request.Header.Set("Content-Type", contentHeader)
 	}
+
+	// Get RemoteAddr from the request
+	clientIP, err := GetClientIP(req)
```

Comment:
thinking about local development. not sure we should do the computation for TIC unless we have provided a TIC Secret.